### PR TITLE
[SRVCOM-2490] Add the FIPS note to 1.29 release notes

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
Serverless 1.29

Issue:
https://issues.redhat.com/browse/SRVCOM-2490

Link to docs preview:
https://60876--docspreview.netlify.app/openshift-serverless/latest/about/serverless-release-notes.html#serverless-rn-1-29-0_serverless-release-notes

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mgencur** [comment]
LGTM from QE

File: modules/serverless-rn-1-29-0.adoc [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 {ServerlessProductName} 1.29 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
 
+[IMPORTANT]
+====
+{ocp-product-title} 4.13 is based on Red Hat Enterprise Linux 9.2, which has not yet been submitted for Federal Information Processing Standard (FIPS) validation. Red Hat expects, though it cannot commit to a specific time, to obtain FIPS validation for RHEL 9.0 and RHEL 9.2 modules, and later even minor releases of RHEL 9.x. Information on updates will be available in the link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards Knowledgebase article].
```
Comment (Srivaralakshmi) [inline modules/serverless-rn-1-29-0.adoc:13]:
```suggestion
{product-title} 4.13 is based on {op-system-base-full} 9.2.  {op-system-base} 9.2 is yet to be submitted for Federal Information Processing Standards (FIPS) validation. Red Hat expects though it cannot commit to a specific timeframe, to obtain FIPS validation for {op-system-base} 9.0 and {op-system-base} 9.2 modules, and later even minor releases of {op-system-base} 9.x. Information on updates will be available in the link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards Knowledgebase article].
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@rh-max Minor suggestions. PTAL. Nice work! Thank you : )

**Review by Srivaralakshmi — APPROVED** [review]
@rh-max Thanks for the changes, Approved!

**Pull request merged**